docs/adapting-new-services.md

```diff
@@ -9,44 +9,44 @@ In order to connect a new service to Harmony:
-6. The service should follow Harmony's recommendations for service implementations
+6. A new Argo workflow template needs to supply the information to invoke the service as part of a workflow
+7. The service should follow Harmony's recommendations for service implementations
+8. The docker image for the service must be published somewhere for harmony to reach
 
 A simple reference service, [harmony-service-example](https://github.com/nasa/harmony-service-example), provides examples of each. The remainder of this document describes how to fulfill these requirements in more depth.
 
 ## 1. Allowing Harmony to invoke services
 
-Harmony provides a Python library, [harmony-service-lib-py](https://github.com/nasa/harmony-service-lib-py), to
-ease the process of adapting Harmony messages to subsetter code.  It provides helpers for message parsing, command line interactions, data staging,
-and Harmony callbacks.  Full details as well as an example can be found in the project's README and code.
+Harmony provides a Python library, [harmony-service-lib-py](https://github.com/nasa/harmony-service-lib-py), to ease the process of adapting Harmony messages to subsetter code. It provides helpers for message parsing, command line interactions, data staging, reading and writing STAC catalogs, and Harmony callbacks. Full details as well as an example can be found in the project's README and code.
 
 ### Docker Container Images
 
-The service and all necessary code and dependencies to allow it to run can be packaged in a Docker container image.  Docker images can be staged anywhere Harmony can reach them, e.g. Dockerhub or AWS ECR.  Harmony will run the Docker image, passing the following command-line parameters:
+The service and all necessary code and dependencies to allow it to run can be packaged in a Docker container image. Docker images can be staged anywhere Harmony can reach them, e.g. Dockerhub or AWS ECR. Harmony will run the Docker image, passing the following command-line parameters:
 
 `--harmony-action <action> --harmony-input <input> --harmony-sources <sources-file>`
 
-`<action>` is the action Harmony wants the service to perform.  Currently, Harmony only uses `invoke`, which requests that the service be run and exit.  The service library Harmony provides also supports a `start` action with parameter `--harmony-queue-url <url>`, which requests that the service be started as a long running service that reads requests from an SQS queue.  This is likely to be deprecated.
+`<action>` is the action Harmony wants the service to perform. Currently, Harmony only uses `invoke`, which requests that the service be run and exit. The service library Harmony provides also supports a `start` action with parameter `--harmony-queue-url <url>`, which requests that the service be started as a long running service that reads requests from an SQS queue. This is likely to be deprecated.
```

Comment:
Could you please do me a favor and move yourself and these single spaces somewhere that is no longer on my lawn?